doc/lua_api.txt

```diff
@@ -2590,6 +2591,37 @@ An `InvRef` is a reference to an inventory.
 * `get_location()`: returns a location compatible to `minetest.get_inventory(location)`
     * returns `{type="undefined"}` in case location is not known
 
+### `AreaStore`
+A fast access data structure to store areas, and find areas near a given position or area.
+Every area has a `data` string attribute to store additional information.
+You can create an empty `AreaStore` by calling `AreaStore()`, or `AreaStore(type_name)`.
+If you chose the parameter-less constructor, a fast implementation will be automatically chosen for you.
+#### Methods
+* `get_area(id, borders, data)`: returns the area with the id `id`. (optional) Boolean values `borders` and `data` control what's copied.
+* `get_areas_for_pos(pos, borders, data)`: returns all areas that contain the position `pos`. (optional) Boolean values `borders` and `data` control what's copied.
+* `get_areas_in_area(area, borders, data)`: returns all areas that contain all nodes inside `area`. If `accept_overlap` is true, also areas are returned that have nodes in common with `area`. (optional) Boolean values `borders` and `data` control what's copied.
```

Comment:
I mean, it's not listed in the argument list in the docs, only area, borders, and data are listed.  It doesn't seem to make any sense to pass (`area` and `borders`) or `data` to this function though.